stdlib/testing/kapacitor/eval_map_with_test.flux

```diff
@@ -33,4 +33,3 @@ test _map = () => ({input: testing.loadStorage(csv: inData), want: testing.loadM
 // stream 
 //  |eval(lambda: 2 * _value)
 //.as('_newValue')
-
```

Comment:
Was this a formatter change?

Reply:
Yeah I assume so, it just stripped an empty line. Seems fine right?

Reply:
Hmm, its a little weird to strip out the final newline in a file. Usually linters prefer that there's at least one newline at the end of the file. But its outside the scope of this pr, so i'd disregard.